server/server.go

```diff
@@ -119,26 +119,26 @@ func (cs *cribbageServer) ginPostCreateGame(c *gin.Context) {
 		return
 	}
 
-	// TODO investigate what it'll take to protobuf-ify our models
-	c.JSON(http.StatusOK, g)
+	c.JSON(http.StatusOK, network.NewCreateGameResponse(g))
```

Comment:
With the changes in this file: can we ensure that we don't regress functionality in the wasm client too? I just merged that and it's the only client we have that "works" today:#